NAMESPACE

```diff
@@ -1,13 +1,16 @@
 # Generated by roxygen2: do not edit by hand
 
 S3method(augment,multinom)
+S3method(augment,xgb.Booster)
```

Comment:
Hmm, even if I write a comment here, devtools::check() removes it. Do you have an idea where to put comments?